editoast/src/models/stdcm_search_environment.rs

```diff
@@ -127,11 +128,31 @@ pub mod tests {
             .work_schedule_group_id(Some(work_schedule_group.id))
             .temporary_speed_limit_group_id(Some(temporary_speed_limit_group.id))
             .timetable_id(timetable.id)
-            .search_window_begin(NaiveDate::from_ymd_opt(2024, 1, 1).unwrap().into())
-            .search_window_end(NaiveDate::from_ymd_opt(2024, 1, 15).unwrap().into());
+            .search_window_begin(
+                NaiveDate::from_ymd_opt(2024, 1, 1)
+                    .unwrap()
+                    .and_hms_opt(0, 0, 0)
+                    .unwrap()
+                    .and_utc(),
```

Comment:
You can replace by this.

```suggestion
                 DateTime::parse_from_rfc3339("2024-01-01T00:00:00Z").unwrap().to_utc()),
```

Reply:
Oh, you found an even better API for that 😍 I’m resolving.